### DISCUSSION TITLE
Buttons are not following the same style

### DISCUSSION THREAD

**Discussion by DigitalGreyHat** [community/community#8621] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/90456651/145476141-581c5142-5142-4351-8e3d-872088fb5ad1.png)
The "Star" button has a vertical line between the text and the arrow. The "Watch" button has no line between the text and the arrow. Looks kinda weird, without the line!

Edit: Just discovered the same is for the buttons in the repo search bar. (maybe there it makes sense)
![image](https://user-images.githubusercontent.com/90456651/145476824-48dd4239-6f28-412c-a36a-3ae08df5ff64.png)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note — this is because in the case of the Watch button, the entire button acts as a dropdown trigger. For the Star/List button, these are divided because each section of the button performs a different action (starring/unstarring, versus opening the lists dropdown menu).

Hopefully that makes sense 😊
